### ISSUE TITLE
have you been able to get this to work?

### ISSUE THREAD

**Issue by dgauche** [monti-tls/Dymo-LT-Application#1]
I would like to print my own graphics like qr codes?

**Comment by monti-tls** [comment]
Yes, it works perfectly with my dymo letratag, i use it routinely to print out electrical panel labels
You can print any graphics so long as it fits in the 32 pixels high QPixmap
